paper.md

```diff
@@ -5,17 +5,20 @@ title: Paper and Analysis
 
 CSP.guru has been developed since 2016 by Johan Lilliestam and his lab. The following publications refer to the data and use it among their own unique contributions for some of their analysis. You are free to use the data for any academic work by citing the version you use, see [Zenodo for details](https://doi.org/10.5281/zenodo.1318151) and one of the publications leading to new csp.guru version for peer-reviewed methodology behind the data. If you want to collaborate on additional publications or have used csp.guru for another publication not listed here and what to get it up on the page, please [get in touch](mailto:richard.thonig@iass-potsdam.de).
 
-<b>Suggested citation:</b> Johan Lilliestam, Richard Thonig, Chuncheng Zang, & Alina Gilmanova. (2021). CSP.guru (Version 2021-01-01) [Data set]. Zenodo. [http://doi.org/10.5281/zenodo.4613099](http://doi.org/10.5281/zenodo.4613099)
+<b>Suggested citation:</b> Johan Lilliestam, Richard Thonig, Chuncheng Zang, & Alina Gilmanova. (2021). CSP.guru (Version 2021-07-01) [Data set]. Zenodo. [http://doi.org/10.5281/zenodo.5094290](http://doi.org/10.5281/zenodo.5094290)
 
 ## Peer-reviewed publications leading to new csp.guru versions
 
 Lilliestam, J., L. Ollier, M. Labordena, S. Pfenninger and R. Thonig (2021). The near- to mid-term outlook for concentrating solar power: mostly cloudy, chance of sun, in: [Energy Sources, Part B: Economics, Planning, and Policy 16 (1): 23-41.](https://doi.org/10.1080/15567249.2020.1773580)
 
 Lilliestam, J. and R. Pitz-Paal (2018). Concentrating solar power for less than USD 0.07 per kWh: finally the breakthrough?, in: [Renewable Energy Focus 26: 17-21](https://doi.org/10.1016/j.ref.2018.06.002)
 
-Lilliestam, J., Labordena, M., Patt, A., & Pfenninger, S. (2017). Empirically observed learning rates for concentrating solar power and their responses to regime change. [Nature Energy, 2, 17094.]( https://doi.org/10.1038/nenergy.2017.94)
+Lilliestam, J., M. Labordena, A. Patt, & S. Pfenninger, (2017). Empirically observed learning rates for concentrating solar power and their responses to regime change. [Nature Energy, 2, 17094.]( https://doi.org/10.1038/nenergy.2017.94)
```

Comment:
This change doesn't make sense, does it?

Reply:
why not? all the other entries have the initial first than the last name, only this one was the other way around?

Reply:
Oh ok, you are right! But are all the others correct? :P

It looks odd to me, but it may well be correct. Not a big issue in any case.

Reply:
Maybe. I just copied them formated out of endnote. In any case. Consistency is key. If they are wrong, they should all be wrong in the same way :)